tests/test_celeba_format.py

```diff
@@ -0,0 +1,72 @@
+from unittest import TestCase
+import os.path as osp
+
+import numpy as np
+
+from datumaro.components.annotation import Bbox, Label, Points
+from datumaro.components.dataset import Dataset
+from datumaro.components.extractor import DatasetItem
+from datumaro.plugins.celeba_format import CelebaImporter
+from datumaro.util.test_utils import compare_datasets
+
+from .requirements import Requirements, mark_requirement
+
+DUMMY_DATASET_DIR = osp.join(osp.dirname(__file__), 'assets',
+    'celeba_dataset')
+
+class CelebaImporterTest(TestCase):
+    @mark_requirement(Requirements.DATUM_475)
+    def test_can_import(self):
```

Comment:
There should be a similar test for the case when only the aligned images are present.